src/time/span.cr

```diff
@@ -286,6 +286,10 @@ struct Time::Span
   def self.zero
     new(0)
   end
+              
+  def zero?
+    self == typeof(self).zero
```

Comment:
That could work, although it seems a bit verbose to have a module just for that.